src/components/EditableTextarea/EditableTextarea.Mask.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react'
```

Comment:
Ignore this file please, I remove it in the next commit -- was a leftover from an initial attempt that accidentally made its way into staging.